### DISCUSSION TITLE
Can't find stylesheet to import

### DISCUSSION THREAD

**Discussion by charlesanim** [fwouts/previewjs#923] [General]
We use multiple scss libs in our components. When I open Preview.js extension, I get a console error that
`Can't find stylesheet to import`. How do I fix this?

Also, is there a config we can specify to import all the scss that our app needs so its available for previewjs to infer?

**Comment by fwouts** [comment]
Hey @charlesanim, have you seen https://previewjs.com/docs/config/css?

**Reply by charlesanim** [reply]
Thanks for the link? Does this mean wrapping every component in this wrapper? Might not be feasible for a larger org repo. Is is possible to pass off a list of imports for Previewjs?

**Reply by fwouts** [reply]
Yes, you could do the following without defining any wrapper component:

```tsx
// __previewjs__/Wrapper.tsx

import "../src/some-stylesheet.scss";
import "../src/some-other-stylesheet.scss";

export {}
```

Would that work for you?

**Reply by charlesanim** [reply]
i'll try that Thank you
